ansible/setup.yml

```diff
@@ -114,6 +114,7 @@
         - awscli
         - ansible
         - boto
+        - boto3
```

Comment:
Is boto needed then, given that there is boto3?

Reply:
yes, unfortunately different libs use different versions.